sigs/orchestration-security/notes/orchestration-security-jun-13-2017-notes.md

```diff
@@ -0,0 +1,55 @@
+# Meeting #1: June 13, 2017 @ 3pm PST 
+ 
+## Attendees:
+- Diogo Monica (Docker)
+- Riyaz Faizullabhoy (Docker)
+- Ying Li (Docker)
+- Aaron Lehmann (Docker)
+- Olivia Barnett (Docker)
+- Lorenzo Fontana - fntlnz
+ 
+## Agenda: 
+- External Secrets Design Doc review.
+- Service Identities Design Doc review.
+
+## PRs:
+- External swarmkit secrets PR review #2239 
+- [ca/node] Maybe update the root CA when renewing the TLS cert  #2238
+ 
+## Notes:
+- Welcome to SIG! (“Special Interest Group”)
+- Three topic focuses:
+  - secrets management and how to integrate with external secrets management systems
+  - service identities: certificates issued by orchestrator to services
+  - entitlements: providing capability and privilege to services
+- Administrivia:
+  - Weekly meeting
+  - Finding a good meeting time - feedback welcome
+  - Can propose discussion items to Agenda section and PRs to review on PRs section: will be approved by SIG moderator for meeting
+  - All interactions in the open, with notes!
+- External Secrets: pluggable way of having external developers contribute implementations to store secrets long-term on external systems, have swarm be “last-mile” delivery
+  - Plugin model:
+    - Swarm-wide plugins aren’t a reality yet - need plugins to only run in managers and have configuration management. Brian Goff working on plugins https://github.com/moby/moby/pull/33575 , Liron working on first implementation of external plugin https://github.com/docker/swarmkit/pull/2239  - please help them!
+  - No orchestrator code should get modified for a plugin
+  - Goal is to have security guarantees from Swarm secrets should extend to external implementations
+  - Vault implementation coming soon with help from CyberArk
+  - Plugins can leverage swarm secrets to secure connections
+- Lorenzo: how can we be sure that secrets doesn’t leak around network / memory while allowing external secrets?  Is this up to the plugin? using swarm secrets to bootstrap secure network connections in the plugins (everything should be encrypted on the network)
+plugins should only be enabled on managers (least privilege)
+- Consider ramfs vs. tmpfs, with size restrictions (mitigating swap)
```

Comment:
Should this be indented as well, since it relates to Lorenzo's question?  Also should we add a bullet point about recommending everyone run with encrypted swap, since that was also discussed the meeting?